Assets/MixedRealityToolkit-Examples/SharingWithUNET/Scripts/NetworkDiscoveryWithAnchors.cs.meta

```diff
@@ -1,8 +1,9 @@
 fileFormatVersion: 2
-guid: 567e2a1ea65534040ac33441ec4bc34b
+guid: 63970e8e6c149264787d003d8583d1fd
```

Comment:
Not sure all these GUID changes are good.
Did these get changed in the master branch?
Could see people getting missing script references on their prefabs and in their scenes.

Reply:
Yeah, that's weird. I'll dig in.

Reply:
Cool, the meta files have all been reverted. Not sure what changed them.